UnityProject/Assets/Resources/Prefabs/Objects/Something.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
"This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes"

"Something.meta" 
- I guess this is pretty much THE definition of "unnecessary .meta" 👍

Reply:
oh yeah, I used that to rebase my branch

Reply:
A joke a day, keeps the *insert* away ;)